src/containers/TemplateBuilder/shared/OperationContextHelpers.tsx

```diff
@@ -194,18 +204,18 @@ export const exportTemplate: TemplateOperationHelper = async (
       name: snapshotName,
       options: templateExportOptionName,
     }),
-    getFilterBody(id),
+    JSON.stringify(getFilterBody(id)),
     setErrorAndLoadingState
   )
 
   return result
 }
 
 export const duplicateTemplate: TemplateOperationHelper = async (
-  { id, snapshotName },
+  { id, snapshotName, templates = {} },
```

Comment:
I'm stringifying this here, rather than inside `getFilterBody()` like it was previously, so I can merge the "templateOptions" into the body as well before stringifying.